Source/JavaScriptCore/inspector/remote/cocoa/RemoteInspectorCocoa.mm

```diff
@@ -346,6 +346,11 @@ static bool canAccessWebInspectorMachPort()
         pushListingsSoon();
 }
 
+void RemoteInspector::notifyCanConnectToWebInspector()
+{
+    notify_post(WIRServiceAvailableNotification);
```

Comment:
what if Web Inspector is not enabled on the device?  i think this would cause us to attempt to create an XPC connection to `com.apple.webinspector` every time we finish initializing a WebProcess

should we instead `WIRServiceAvailabilityCheckNotification` to ask `com.apple.webinspector` to wake up, which would then send back a `WIRServiceAvailableNotification` if enabled?

Reply:
Ah, that's a good catch! I have addressed this in the latest patch by checking whether we got a Web Inspector sandbox extension from the UI process. We only receive this sandbox extension if Web Inspector is enabled. What do you think?

Thanks for reviewing!